config/locales/en/lookup.yml

```diff
@@ -71,8 +71,8 @@ en:
         heading: From %{date}
         guidance_label: What you can and cannot do from %{date}
         level_one:
-          alert_level: "From %{date} this area will be in Tier 1: Medium alert."
+          alert_level: "%{area} will be in Tier 1: Medium alert."
         level_two:
-          alert_level: "From %{date} this area will be in Tier 2: High alert."
+          alert_level: "%{area} will be in Tier 2: High alert."
```

Comment:
This has a full stop but `alert_level` doesn't for the other tiers

Reply:
I'm not sure what this means, however it looks like this content is being duplicated on line 43, so I'll DRY-up it.